backend/app/api/v1/admin.py

```
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+
+from app.schemas.admin import AdminCreateUser, AdminUserOut, AdminUpdateUser
+from app.services import admin_service
+
+router = APIRouter(prefix="/admin", tags=["admin"])
+
+@router.post("/create_user", response_model=AdminUserOut, status_code=status.HTTP_201_CREATED)
+async def create_user(user_data: AdminCreateUser):
+    try:
+        user = await admin_service.create_user(user_data)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)